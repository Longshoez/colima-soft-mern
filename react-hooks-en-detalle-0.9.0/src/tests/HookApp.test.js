import { shallow } from 'enzyme';
import React from 'react'
import {HookApp} from '../HookApp'

describe('Pruebas en HookAppp', () => {
  
  test('show component correctly', () => {

    const wrapper = shallow(<HookApp />)

    expect(wrapper).toMatchSnapshot()


  });
});
