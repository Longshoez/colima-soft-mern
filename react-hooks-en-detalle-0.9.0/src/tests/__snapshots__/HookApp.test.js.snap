// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HookAppp show component correctly 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
